_layouts/default.html

```diff
@@ -1,6 +1,17 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
+    {% if site.google_analytics_v4 %}
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics_v4 }}"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', '{{ site.google_analytics_v4 }}');
+    </script>
+    {% endif %}
```

Comment:
@f0rked good implementation allowing both support!!

Remember that to pass W3C validation the meta charset must be in the first 1024 characters of page.

Can you move the scripts blocks below, for example at end of head tag?

Reply:
Done, the Google script have to be the first one so I moved all the meta tags to the beginning